browser/src/view/components/Button/Button.scss

```diff
@@ -1,27 +1,26 @@
 @use '../../../styles/common/variables' as *;
 
-.btn-primary {
-  height: 48px;
-  width: 48px;
+.button {
+  font-family: 'Inter';
```

Comment:
Do you really need to have a font definition in a button?  It seems like you can let the button benefit from CSS's cascading rule application.

Reply:
I must not have done this correctly then because I tried that before but it wasn't working. I'll try again.